src/cargo/ops/registry/publish.rs

```diff
@@ -210,34 +258,28 @@ pub fn publish(ws: &Workspace<'_>, opts: &PublishOpts<'_>) -> CargoResult<()> {
 fn wait_for_publish(
```

Comment:
Oh, we are waiting until the first set of packages become available.

We should probably rename this function to clarify that.

Reply:
`wait_for_any_publish_confirmation`